addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1,4 +1,7 @@
 # Kodi Media Center language file
+# [Capitalization] Avoid capitalizing every second word e.g. "This New String", unless proper name or proper noun.
```

Comment:
An "acceptable string" is one where the wording as well as the context it's used for are matching. So a string duplicate is fine as long as the context differs. Not that the context is not about for what it's used (button label, context menu item) but rather in which purpose.
Also, if possible, the contextual note should be written in a generic way and not too specific for the current usage, like "Label of a menu item, button etc that triggers a delete action" instead of "Button label for a dialog to delete a song from the library"